tsconfig.json

```diff
@@ -1,21 +1,13 @@
 {
+  "extends": "astro/tsconfigs/strict",
```

Comment:
LGTM!

One question: how far are we from being able to use `strictest`?

Reply:
38 errors away, 36 of which are caused by `strictNullChecks: true` 😄